### PR TITLE
EZP-28929: User is unable to add translation to Content item

### PR DESCRIPTION
**Pull request by alongosz**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-28929](https://jira.ez.no/browse/EZP-28929)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | TBD
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

This PR fixes inability to add new Translation to a Content item by providing missing Content edit template.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Controller/ContentEditController.php [hunk 1]
```diff
@@ -65,6 +65,10 @@ public function __construct(
      * @param int|null $locationId
      *
      * @return ContentEditView|Response
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
```
Comment (Nattfarinn) [inline src/bundle/Controller/ContentEditController.php]: This is not related to this ticket. Please create new PR with this change.

Comment (webhdx) [reply src/bundle/Controller/ContentEditController.php]: I'd also advise creating new JIRA ticket for this change.

Comment (webhdx) [reply src/bundle/Controller/ContentEditController.php]: Describing how QA can test this change wouldn't harm either. Ah Andrew... I was expecting more from you :/

Comment (Nattfarinn) [reply src/bundle/Controller/ContentEditController.php]: If we really throw exceptions we don't catch, I guess we should document it. /cc: @DominikaK

Comment (sunpietro) [reply src/bundle/Controller/ContentEditController.php]: i-dont-believe-you

Comment (Nattfarinn) [reply src/bundle/Controller/ContentEditController.php]: And I don't like the font.

**Comment by alongosz** [comment]
:joy: :joy: Thank you guys for the review, I've learned a lot!

**Comment by alongosz** [comment]
Thanks @m-tyrala
@lserwatka this is mergable :)
